_posts/2023-02-20-example-mapping-in-practice.markdown

```diff
@@ -0,0 +1,179 @@
+
+To me, example mapping is the closest you can get to perfection. This article sums up the principles of example mapping, compares the opposition, and provides examples itself to help you master it in your own teams.
+
+---
+
+# What is Example Mapping
+
+Example Mapping is a technique using coloured cards to facilitate a session with the dev team (or at least one representative from) and Product Owner and SME(s) to elaborate a user story and create a common understanding amongst the team.
+
+Yellow cards represent the User story/feature, blue cards specify rules, green cards provide examples and red cards are for asking questions.
+
+For this post we are going to be working on a fictional app we are building based on the 'Kwik-E-Mart' from everyone's favourite (well if it's not then it should be) cartoon TV Show 'The Simpsons'.
+
+Apu (the store owner) has decided to go digital to boost business and support his growing family.
+
+Apu kickoffs the session by presenting the requirement:
+
+_Discounts are offered based on the previous month's spend_
+
+Apu then offers up two rules he has already thought of and places them on the canvas on blue cards like so:
+![Example Map](/img/blog/examplemapping/initialrules.png)
+
+The team get about asking questions aroundf the discounts and limit:
+![Example Map](/img/blog/examplemapping/questions.png)
+
+The questions lead to examples being specified, and a new rule emerges which needs examples to help explain:
+
+![Example Map](/img/blog/examplemapping/examplemap.png)
+
+To recap the process. The Product Owner brings a new User Story to the session, presents to the team with a brief explanation and then the facilitator sets to work . I like to run my sessions as follows:
+
+1. Allow the Product Owner to describe the User Story including the how and why
+2. Set a timer for five minutes quiet time and allow the team to raise questions ready for the Product Owner
+3. During the quiet time the Product Owner may start creating Rules on the blue cards
+4. Once quiet time is over I go through the question cards and get the team member who raised it to ask the Product Owner the question
+5. As questions are answered, new rules are defined and then clarified using Examples as a whole team exercise
+6. The board is continually refined as questions are answered, new rules written and examples provided
+7. To ensure that we have something solid to start on, I try to keep focus on one Rule at a time to ensure that sufficient examples are required and continually ask "Dev team, are you happy you can start building this" and "Product Owner, are you happy with this outcome"
+
+At the end of the session the goal is to at least have a starting point for the User Story development and a plan to follow up any additional sessions requried.
+
+<br />
+
+# The Opposition
+
+There's obviouslt many ways to specify and communicate requirements, so lets stack Example Mapping up against the opposition.
```

Comment:
spelt obviously wrong. Though not sure if use of such wording will come across as condescending lol.